### DISCUSSION TITLE
Using "Permanent password" v1.2.1

### DISCUSSION THREAD

**Discussion by jpquinta** [rustdesk/rustdesk#5003] [General]
Hi,
I would like to report one issue on version 1.2.1, that is related when we are trying to use a “permanent password”.
The system does not accept login sessions with the permanent password activated.
I have changed the settings to do the sessions in different modes (accept sessions via both/or only via password), we can only login with the dynamic password generated by the system, the personal password created never accept.

Does anyone have this behavior?
![2023-07-14_11h21_43](https://github.com/rustdesk/rustdesk/assets/42496083/9356afa8-62b1-4cbc-af1f-dc5d8f100fc3)

Thank you
JP

**Comment by 21pages** [comment]
In the dialog for setting permanent password,  the shown password length the same as  you set, or much longer than you set?

**Comment by rustdesk** [comment]
Working in my test anyway.

**Comment by jpquinta** [comment]
Hi all,
Thank you for your feedback.
I have discovered the issue, was a problem with the region of the keyboard.. 🤦‍♂️ 
It was nice to have the option to see the password, 😊
![2023-07-14_11h57_41](https://github.com/rustdesk/rustdesk/assets/42496083/c7c9ab0f-16a2-41fd-843d-30f121f6fe3d)

Again, thank you for your confirmation.
All the best
Cheers
JP

**Comment by disarticulate** [comment]
This is not available in 2023-11-22 V1.2.3-2 in the android build.
